apps/web/utils/navitems.ts

```diff
@@ -0,0 +1,15 @@
+// TODO: The about and contact page
+export const navItems = [
+	{
+		label: "Home",
+		href: "/"
+	},
+	{
+		label: "About",
+		href: "/about"
+	},
+	{
+		label: "Contact",
+		href: "/contact"
```

Comment:
these pages haven't been implemented yet, for now just have home and "my reviews" which redirects to `/my-reviews`